src/main/clm_varpar.F90

```diff
@@ -145,6 +147,8 @@ subroutine clm_varpar_init(actual_maxsoil_patches, actual_numcft)
     cft_lb = natpft_ub + 1
     cft_ub = cft_lb + cft_size - 1
 
+    mxharvests = mxgrowseas + 1
+
```

Comment:
Curious about this. Why could there be 2 harvests for 1 growing season? Could/should these variables be equal instead (e.g., `mxharvests=mxgrowseas`) or is there a reason it needs to be +1 that I don't currently understand?

Reply:
@danicalombardozzi `mxgrowseas` actually refers to the maximum number of sowings. A patch could have 2 harvests in a year even if only 1 sowing is allowed; for instance:
- Planted Oct. 1, 2022 and harvested Jan. 3, 2023
- Planted Oct. 1, 2023 and harvested Dec. 30, 2023

In this case, there are 2 harvests in 2023. Does that make sense?

Confusion might have arisen from the variable name here. I've actually been meaning to rename `mxgrowseas` to `mxsowings` to make this clearer.

Reply:
Thanks for the explanation. Yes, I think renaming this variable might be clearer. Would it be useful to add a comment similar to "Allows for multiple harvests in cases when harvests happen near the beginning or end of a year"?

Reply:
Okay, in commit 7a2c816 I renamed it and added an explanatory comment.